lib/rspec/matchers/built_in/raise_error.rb

```diff
@@ -46,6 +46,7 @@ def matches?(given_proc, negative_expectation=false, &block)
               @raised_expected_error = true
               @with_expected_message = verify_message
             end
+            true
```

Comment:
I don't see what this line accomplishes.  This isn't a return statement, so this doesn't cause `true` to be returned.  There's no variable assignment on the `begin/end` expression that would get the value `true`.  As far as I can see, this does nothing...am I missing something?